### PR TITLE
Support for random banners.

### PR DESCRIPTION
**Pull request by DevelopmentCool2449**
Hello, this patch allow to users display random banners in a list.
I'd like to hear their reviews.

### CODE REVIEW

**Comment by jcs090218** [comment]
One warning:

```
dashboard-widgets.el:348:2: Warning: custom-declare-variable ‘dashboard-startup-banner’ docstring has wrong usage of unescaped single quotes (use \= or different quoting)
```

File: dashboard-widgets.el [hunk 1]
```diff
@@ -740,7 +758,7 @@ Argument IMAGE-PATH path to the image."
       (when graphic-mode (insert "\n"))
       ;; If specified, insert a text banner.
       (when-let ((txt (plist-get banner :text)))
-        (if (eq dashboard-startup-banner 'ascii)
+        (if (length= banner 2)
```
Comment (jcs090218) [inline dashboard-widgets.el:761]: This looks weird. Try avoid the magic number here. 🤔

**Comment by DevelopmentCool2449** [comment]
> One warning:
> 
> ```
> dashboard-widgets.el:348:2: Warning: custom-declare-variable ‘dashboard-startup-banner’ docstring has wrong usage of unescaped single quotes (use \= or different quoting)
> ```

Fixed.

File: dashboard-widgets.el (hunk 1 above)
Comment (DevelopmentCool2449) [reply dashboard-widgets.el:761]: > This looks weird. Try avoid the magic number here. 🤔

That was the only way i found to check if banner is a cons cell, using listp or consp conflicts with random banner list.

Comment (DevelopmentCool2449) [reply dashboard-widgets.el:761]: If you know a way to determine a cons cell, i'd really appreciate if you could tell me.

Comment (DevelopmentCool2449) [reply dashboard-widgets.el]: Fixed, only determining if it is a list fixs this issue.

**Comment by jcs090218** [comment]
It seems like it doesn't work inside the terminal. 🤔

**Comment by DevelopmentCool2449** [comment]
> It seems like it doesn't work inside the terminal. 🤔 

Fixed

**Review by jcs090218 — APPROVED** [review]

**Comment by jcs090218** [comment]
Thank you!

**Pull request merged**